src/app/routes/main/components/tech-picker/tech-picker.component.ts

```diff
@@ -114,13 +114,14 @@ export class TechPickerComponent {
   }
 
   copyScriptToClipboard(): void {
-    const script = `/c local list = {}
-for _, tech in pairs(game.player.force.technologies) do 
+    const script = `local list = {}
```

Comment:
This should still include `/c`, correct? Otherwise when pasted the command does not run?

Reply:
You're right, one second please!

Reply:
Thanks, I'll test this out the next chance I get, but it looks good to me